src/pages/index.tsx

```diff
@@ -1,5 +1,20 @@
+import { GetStaticProps } from 'next'
 import BaseNewLayout from 'layouts/base-new'
 import HomePageView from 'views/homepage'
+import { generateStaticProps } from 'views/homepage/server'
+
+export const getStaticProps: GetStaticProps = async () => {
+  return generateStaticProps({
+    featuredLearnContent: [
+      {
+        collectionSlug: 'vault/associate-cert',
+      },
+      {
+        collectionSlug: 'vault/ops-pro-cert',
```

Comment:
We should probably adjust all the data in this view to be editable via a json file, to follow similar patterns. I'm just thinking of authors needing to come and update content in this file, whereas an abstracted data file would be ideal. Of course, this is a long term thought, [not necessary for this PR. ](https://app.asana.com/0/1202276096167515/1202445483168876)